swcc/tests/test_download_upload.py

```diff
@@ -26,8 +26,10 @@ def project_as_dict_repr(project):
     del project_repr['dataset']['id']
     del project_repr['dataset']['creator']
     del project_repr['last_cached_analysis']
-    del project_repr['last_cached_deep_ssm']
     del project_repr['landmarks_info']
+
+    if 'last_cached_deep_ssm' in project_repr:
+        del project_repr['last_cached_deep_ssm']
```

Comment:
is this a store variable?

Reply:
No, this is referring to a key in the project serializer. Your PR has a statement to remove this key before comparing the serialized remote project to the serialized local project. This statement was getting a `KeyError`, so I wrapped that statement with an `if` clause.

Reply:
Correction: Rather than serializer, I mean the output of `dict(project)`

Reply:
Git blame has informed me that it was myself, 4 months ago, who added that... The model has since changed so project has no reference to `last_cached_deepssm` (or any deepssm value for that matter). This can be removed here.

Reply:
Ok, that makes sense. I removed those lines in [8e38ac0](https://github.com/girder/shapeworks-cloud/pull/363/commits/8e38ac014196c9b8360c9ce8ea7411eebe7c32cc)